src/utils/getPriceGuidance.ts

```diff
@@ -0,0 +1,45 @@
+import { reduce } from "lodash"
+import metaphysics from "../lib/metaphysics"
+const currency = require("currency.js")
+
+const formatCurrency = value => currency(value, { separator: "" }).format()
```

Comment:
I'm curious to know what the saved value looks like? From here it appears to be `1000` -- but also curious if the averages do floats `1000.95` etc.

Reply:
I'm rounding floats up if necessary on line [44](https://github.com/artsy/kaws/blob/cb48af66868cd40a1af4889e1001da834403daea/src/utils/getPriceGuidance.ts#L44).  So in that case `1000.95` woule be `1001`.